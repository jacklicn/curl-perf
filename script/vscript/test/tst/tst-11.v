

sub foo3(a,b,c)
    return aio_inc( b )
end

sub foo2(z,y)
    foo3( z * (z - 1), z, 3 * y)
end

sub foo(x)
    foo2( x - 1, x + 1 )
end

print( foo(3) )
