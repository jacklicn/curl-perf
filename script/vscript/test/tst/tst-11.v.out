*** syntax ok ***
*** semantics and code generation ok ***

>3
